swarm_copy/tools/morphology_features_tool.py

```diff
@@ -0,0 +1,147 @@
+"""Morphology features tool."""
+
+import logging
+from typing import Any, ClassVar
+
+import neurom
+import numpy as np
+from neurom import load_morphology
+from pydantic import BaseModel
+
+from swarm_copy.tools.base_tool import BaseMetadata, BaseTool
+from swarm_copy.utils import get_kg_data
+
+logger = logging.getLogger(__name__)
+
+
+class MorphologyFeatureOutput(BaseModel):
+    """Output schema for the neurom tool."""
+
+    brain_region: str
+    feature_dict: dict[str, Any]
+
+
+class MorphologyFeaturesInput(BaseModel):
+    """Inputs for MorphologyFeatureTool."""
+
+    morphology_id: str
```

Comment:
There should be a `Field(description=...)` here.

Reply:
Should I add some description or other configuration for the Field ?

Reply:
You can copy the `Field` in the mirror class in `neuroagent`, i.e.
```python
morphology_id: str = Field(
        description=(
            "ID of the morphology of interest. A morphology ID is an HTTP(S) link, it"
            " should therefore match the following regex pattern:"
            r" 'https?://\S+[a-zA-Z0-9]'"
        )
    )
   ```